Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,118 @@
+# Motor System
+
+## Subsystem Function
+
+This subsystem's role in our design is to move new devices to the checkout door by receiving power from our power subsystem and rotating the platforms holding the devices. This will only be done when the machine senses that there is a device missing from the checkout drawer. 
+
+## Constraints
```

Comment:
The first step is to list the constraints on the subsystem. Without this further feedback is really not useful. State the precise set of constraints and specifications for the subsystem, show your buildable design artifacts, explain how you analyzed it to show that it will meet the constraints.